examples/react-native/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
You can drop this `.eslintrc`, the folder already contains one that inherits from the top level configuration.

Reply:
It is necessary because https://github.com/algolia/react-instantsearch/pull/2304#issuecomment-483241937.

Reply:
I don't think so. The issue was present because we added an `.eslintrc` during the migration to new template. The previous example didn't have an `.eslintrc` but it fallbacks on the one inside the parent folder.